src/main/java/org/folio/settings/server/storage/SettingsStorage.java

```diff
@@ -341,8 +341,8 @@ public Future<Void> getEntries(HttpServerResponse response, String cqlQuery,
 
     PgCqlDefinition definition = PgCqlDefinition.create();
     definition.addField("id", new PgCqlFieldUuid());
-    definition.addField("scope", new PgCqlFieldText());
-    definition.addField("key", new PgCqlFieldText());
+    definition.addField("scope", new PgCqlFieldText().withExact());
```

Comment:
Again, just my curiosity, but why does the scope field need `withExact()` when the id and userId fields do not?

Reply:
The text handling may have rely on B-tree or GIN (full-text) or both. We don't enable either in the construction, but one or two of these should be enabled. This too was changed recently. It does not apply to other fields as they always rely on B-tree.